assets/js/common.js

```diff
@@ -25,7 +26,15 @@ function createTable({
 
     return `0000${major}`.slice(-4) + `0000${minor}`.slice(-4) + `0000${patch}`.slice(-4)
   }
-
+  const ajax = ajaxUrl && {
```

Comment:
This will still error because you're passing it to the `extend` function on line 80 and will basically be passing it undefined if it doesn't exist. Ideally if there is no URL it should report an error and then return from the function.

Reply:
it now uses either data or ajax so it works, there's no errors. I guess it would ideally throw an error if neither were supplied